src/frontend/src/layouts/PageTitle/PageTitle.tsx

```diff
@@ -27,17 +27,22 @@ const PageTitle: React.FC<PageTitleProps> = ({ title, previousPages, headerRight
     <>
       <PageBreadcrumbs currentPageTitle={title} previousPages={previousPages} />
       <Box sx={{ mb: 2 }}>
-        <Grid container>
-          <Grid item xs={6} md={8}>
+        <Grid container rowSpacing={1}>
```

Comment:
also why are tabs a react element, shouldnt they be NERTabs

Reply:
?, you can't make the prop of type NERTabs, the type is ReactElement